Source/Core/Core/HW/DSPHLE/UCodes/AXVoice.h

```diff
@@ -126,8 +148,12 @@ void WritePB(Memory::MemoryManager& memory, u32 addr, const PB_TYPE& pb, u32 crc
 {
   if (HasLpf(crc))
   {
+#ifdef __AVX2__
+    CopyPBSwapped(memory.GetPointerForRange(addr, sizeof(pb)), &pb);
+#else
     const u16* src = (const u16*)&pb;
     memory.CopyToEmuSwapped<u16>(addr, src, sizeof(pb));
+#endif
```

Comment:
What do you think about adding this as a specialization of CopyToEmuSwapped instead? That way the ifdefs can be in just one location, in Common.